src/components/Header/index.js

```diff
@@ -0,0 +1,24 @@
+import { FaGithub } from 'react-icons/fa';
+import styles from './styles.module.scss';
+
+export function Header() {
+  return (
+    <header className={styles.header}>
+      <div className={styles.containerHeader}>
+        <img
+          src="/brasilapi-logo-medium.png"
+          className="logoImg"
```

Comment:
Não deveria ser ao invés de pegar uma classe "global":
```suggestion
          className={styles.logoImg}
```